source/2020-05-15-the-ember-times-issue-148.md

```diff
@@ -55,14 +57,19 @@ Check out the full video explaining the `{{on}}` modifier on [EmberMap](https://
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Ember performance tweaks 🏎](https://abhilashlr.in/ember-performance-tweaks-part-1)
+  
+[Abhilash LR (@abhilashlr)](https://github.com/abhilashlr) blogs about **optimising build timelines & bundle size** in your Ember apps, the first in a series on getting started with performance optimizations! Check out his tips and tricks in the writeup [here](https://abhilashlr.in/ember-performance-tweaks-part-1). Highlights include:
```

Comment:
I noticed that you used `z`  for other words with s/z variance so I think it'd be good to use the American spelling consistently.

```suggestion
[Abhilash LR (@abhilashlr)](https://github.com/abhilashlr) blogs about **optimizing build timelines & bundle size** in your Ember apps, the first in a series on getting started with performance optimizations! Check out his tips and tricks in the writeup [here](https://abhilashlr.in/ember-performance-tweaks-part-1). Highlights include:
```

Reply:
@ijlee2 aha, good catch!